CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/suppor
 2.4.0 (in progress)
 ===================
 
+New Features
+------------
+
+- Added `NetworkMonitor` class and `networkMonitor` property to `connect.options`. This property monitors network information and reconnects participants.
```

Comment:
lets update this to say something like:
' Added a new experimental `connectOptions` property `networkMonitor` which when specified to `true` would use signal from browsers's api to detect changes in networks. Specificing this will result in faster reconnection.

Reply:
@PikaJoyce This class and the ConnectOptions flag are not customer facing. We should change this to something like:
```
- twilio-video.js now supports faster signaling reconnections due to network disruption or handoff. (JSDK-2739)
- twilio-video.js now supports faster media reconnections due to network disruption or handoff. (JSDK-2742)
```